main/Resource.rdef

```diff
@@ -0,0 +1,131 @@
+
```

Comment:
Remove one empty line. For some reason many rdef files have one empty line at the top, no idea why...

Reply:
I noticed it too. It must be an idiosyncrasy of `rc`.

Reply:
Okay! I fixed the three things, let me know if it's okay :)

Reply:
Looks perfect now. :)
@humdingerb Do you want an empty line at the top?

Reply:
Before there were 2 empty lines at the top. As you said, 1 empty line is encountered often for rsrc-converted files. I draw the line at two empty lines... :)
No empty lines is fine by me. Let's merge!
Thanks @claire50!

Reply:
Awesome thank you!!